src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderAssessmentResource.java

```diff
@@ -61,6 +61,14 @@
             @ApiResponse(code = 200, message = "The CSRA assessment list is returned, 1 per offender.", response = Assessment.class, responseContainer = "List")})
     List<Assessment> postOffenderAssessmentsCsraList(@ApiParam(value = "The required offender numbers (mandatory)", required = true) @NotEmpty @RequestBody List<String> body);
 
+    @GetMapping("/assessments")
+    @ApiOperation(value = "Returns assessment information on Offenders at a prison.")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "OK", response = Assessment.class, responseContainer = "List")})
```

Comment:
Maybe add the 400 bad request for when the `validateOffenderList` throws?

Reply:
good idea